eos/b-decays/b-to-dstar-l-nu.hh

```diff
@@ -39,25 +39,41 @@ namespace eos
 
             // Differential Observables
             double differential_branching_ratio(const double & s) const;
+            double differential_a_fb_leptonic(const double & s) const;
+            double differential_ratio_tau_mu(const double & s) const;
 
             // Differential Observables - normalized(|Vcb|=1)
             double normalized_differential_branching_ratio(const double & s) const;
 
+            // Four Differential Observables - normalized(|Vcb|=1)
```

Comment:
Please call it

  normalized_four_differential_decay_width

to remind the user that |Vcb|=1 !

Reply:
done! and i think the PR should now be ready for merge. cheers